02-16 21:38:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Unknown data type: "FK_TERAPIA_PAZIENTE_IDX"; SQL statement:
CREATE TABLE IF NOT EXISTS `Terapia` (
  `ID` INT NOT NULL,
  `Descrizione` VARCHAR(255) NULL,
  `DataInizio` DATE NULL,
  `DataFine` DATE NULL,
  `Paziente_CODF` VARCHAR(255) NOT NULL,
  PRIMARY KEY (`ID`),
  INDEX `fk_Terapia_Paziente_idx` (`Paziente_CODF` ASC),
  CONSTRAINT `fk_Terapia_Paziente`
    FOREIGN KEY (`Paziente_CODF`)
    REFERENCES `mydb`.`Paziente` (`CODF`)
    ON DELETE NO ACTION
    ON UPDATE NO ACTION) [50004-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:4000)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:3863)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5771)
	at org.h2.command.Parser.parseCreate(Parser.java:4159)
	at org.h2.command.Parser.parsePrepared(Parser.java:350)
	at org.h2.command.Parser.parse(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:277)
	at org.h2.command.Parser.prepareCommand(Parser.java:242)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
02-16 21:39:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Schema "MYDB" not found; SQL statement:
CREATE TABLE IF NOT EXISTS `Terapia` (
  `ID` INT NOT NULL,
  `Descrizione` VARCHAR(255) NULL,
  `DataInizio` DATE NULL,
  `DataFine` DATE NULL,
  `Paziente_CODF` VARCHAR(255) NOT NULL,
  PRIMARY KEY (`ID`),
  CONSTRAINT `fk_Terapia_Paziente`
    FOREIGN KEY (`Paziente_CODF`)
    REFERENCES `mydb`.`Paziente` (`CODF`)
    ON DELETE NO ACTION
    ON UPDATE NO ACTION) [90079-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.Parser.getSchema(Parser.java:656)
	at org.h2.command.Parser.getSchema(Parser.java:663)
	at org.h2.command.Parser.parseReferences(Parser.java:5683)
	at org.h2.command.Parser.parseAlterTableAddConstraintIf(Parser.java:5657)
	at org.h2.command.Parser.parseCreateTable(Parser.java:5765)
	at org.h2.command.Parser.parseCreate(Parser.java:4159)
	at org.h2.command.Parser.parsePrepared(Parser.java:350)
	at org.h2.command.Parser.parse(Parser.java:305)
	at org.h2.command.Parser.parse(Parser.java:277)
	at org.h2.command.Parser.prepareCommand(Parser.java:242)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1189)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1390)
	at org.h2.server.web.WebApp.query(WebApp.java:1063)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1025)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1012)
	at org.h2.server.web.WebThread.process(WebThread.java:168)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
02-16 21:40:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "INFERMIERE" not found; SQL statement:
CREATE TABLE IF NOT EXISTS `TerInfCross` (
  `Terapia_ID` INT NOT NULL,
  `Infermiere_CODF` VARCHAR(255) NOT NULL,
  PRIMARY KEY (`Terapia_ID`, `Infermiere_CODF`),
  CONSTRAINT `fk_Terapia_has_Infermiere_Terapia1`
    FOREIGN KEY (`Terapia_ID`)
    REFERENCES `Terapia` (`ID`)
    ON DELETE NO ACTION
    ON UPDATE NO ACTION,
  CONSTRAINT `fk_Terapia_has_Infermiere_Infermiere1`
    FOREIGN KEY (`Infermiere_CODF`)
    REFERENCES `Infermiere` (`CODF`)
    ON DELETE NO ACTION
    ON UPDATE NO ACTION) [42102-187]
02-16 22:06:48 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Cannot parse "DATE" constant "01/02/2016"; SQL statement:
INSERT INTO "PUBLIC"."TERAPIA"("ID","DESCRIZIONE","DATAINIZIO","DATAFINE","PAZIENTE_CODF")VALUES(?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5) [22007-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:860)
	at org.h2.table.Column.convert(Column.java:148)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:157)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:143)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:322)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:2966)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1085)
	at org.h2.server.web.WebApp.process(WebApp.java:227)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: 01/02/2016
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:275)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 16 more
02-16 22:06:48 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Cannot parse "DATE" constant "01/02/2016"; SQL statement:
INSERT INTO "PUBLIC"."TERAPIA"("ID","DESCRIZIONE","DATAINIZIO","DATAFINE","PAZIENTE_CODF")VALUES(?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5) [22007-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:860)
	at org.h2.table.Column.convert(Column.java:148)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:157)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:143)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:322)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:2966)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1085)
	at org.h2.server.web.WebApp.process(WebApp.java:227)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: 01/02/2016
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:275)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 16 more
02-16 22:07:43 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Cannot parse "DATE" constant "01-02-2016"; SQL statement:
INSERT INTO "PUBLIC"."TERAPIA"("ID","DESCRIZIONE","DATAINIZIO","DATAFINE","PAZIENTE_CODF")VALUES(?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5) [22007-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:860)
	at org.h2.table.Column.convert(Column.java:148)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:157)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:143)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:322)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:2966)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1085)
	at org.h2.server.web.WebApp.process(WebApp.java:227)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: 1-2-2016
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:281)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 16 more
02-16 22:07:43 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Cannot parse "DATE" constant "01-02-2016"; SQL statement:
INSERT INTO "PUBLIC"."TERAPIA"("ID","DESCRIZIONE","DATAINIZIO","DATAFINE","PAZIENTE_CODF")VALUES(?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5) [22007-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:860)
	at org.h2.table.Column.convert(Column.java:148)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:157)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:143)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:322)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:2966)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1085)
	at org.h2.server.web.WebApp.process(WebApp.java:227)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: 1-2-2016
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:281)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 16 more
02-16 22:08:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Cannot parse "DATE" constant "01"; SQL statement:
INSERT INTO "PUBLIC"."TERAPIA"("ID","DESCRIZIONE","DATAINIZIO","DATAFINE","PAZIENTE_CODF")VALUES(?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5) [22007-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:860)
	at org.h2.table.Column.convert(Column.java:148)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:157)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:143)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:322)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:2966)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1085)
	at org.h2.server.web.WebApp.process(WebApp.java:227)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: 01
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:275)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 16 more
02-16 22:08:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Cannot parse "DATE" constant "01"; SQL statement:
INSERT INTO "PUBLIC"."TERAPIA"("ID","DESCRIZIONE","DATAINIZIO","DATAFINE","PAZIENTE_CODF")VALUES(?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5) [22007-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:860)
	at org.h2.table.Column.convert(Column.java:148)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:157)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:143)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:322)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:2966)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1085)
	at org.h2.server.web.WebApp.process(WebApp.java:227)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: 01
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:275)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 16 more
02-16 22:08:14 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Cannot parse "DATE" constant "01"; SQL statement:
INSERT INTO "PUBLIC"."TERAPIA"("ID","DESCRIZIONE","DATAINIZIO","DATAFINE","PAZIENTE_CODF")VALUES(?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5) [22007-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:860)
	at org.h2.table.Column.convert(Column.java:148)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:157)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:143)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:322)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:2966)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1085)
	at org.h2.server.web.WebApp.process(WebApp.java:227)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: 01
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:275)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 16 more
02-16 22:08:14 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Cannot parse "DATE" constant "01"; SQL statement:
INSERT INTO "PUBLIC"."TERAPIA"("ID","DESCRIZIONE","DATAINIZIO","DATAFINE","PAZIENTE_CODF")VALUES(?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5) [22007-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:860)
	at org.h2.table.Column.convert(Column.java:148)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:157)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:143)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:322)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:2966)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1085)
	at org.h2.server.web.WebApp.process(WebApp.java:227)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: 01
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:275)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 16 more
02-16 22:08:26 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Cannot parse "DATE" constant "01/02/2016"; SQL statement:
INSERT INTO "PUBLIC"."TERAPIA"("ID","DESCRIZIONE","DATAINIZIO","DATAFINE","PAZIENTE_CODF")VALUES(?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5) [22007-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:860)
	at org.h2.table.Column.convert(Column.java:148)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:157)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:143)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:322)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:2966)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1085)
	at org.h2.server.web.WebApp.process(WebApp.java:227)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: 01/02/2016
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:275)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 16 more
02-16 22:08:26 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Cannot parse "DATE" constant "01/02/2016"; SQL statement:
INSERT INTO "PUBLIC"."TERAPIA"("ID","DESCRIZIONE","DATAINIZIO","DATAFINE","PAZIENTE_CODF")VALUES(?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5) [22007-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:860)
	at org.h2.table.Column.convert(Column.java:148)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:157)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:143)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:322)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:2966)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1085)
	at org.h2.server.web.WebApp.process(WebApp.java:227)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: 01/02/2016
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:275)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 16 more
02-16 22:09:30 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Cannot parse "DATE" constant "20160201"; SQL statement:
INSERT INTO "PUBLIC"."TERAPIA"("ID","DESCRIZIONE","DATAINIZIO","DATAFINE","PAZIENTE_CODF")VALUES(?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5) [22007-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:860)
	at org.h2.table.Column.convert(Column.java:148)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:157)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:143)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:322)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:2966)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1085)
	at org.h2.server.web.WebApp.process(WebApp.java:227)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: 20160201
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:275)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 16 more
02-16 22:09:30 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Cannot parse "DATE" constant "20160201"; SQL statement:
INSERT INTO "PUBLIC"."TERAPIA"("ID","DESCRIZIONE","DATAINIZIO","DATAFINE","PAZIENTE_CODF")VALUES(?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5) [22007-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:860)
	at org.h2.table.Column.convert(Column.java:148)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:157)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:143)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:322)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:2966)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1085)
	at org.h2.server.web.WebApp.process(WebApp.java:227)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: 20160201
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:275)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 16 more
02-16 22:10:14 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Cannot parse "DATE" constant "01-mar-2016"; SQL statement:
INSERT INTO "PUBLIC"."TERAPIA"("ID","DESCRIZIONE","DATAINIZIO","DATAFINE","PAZIENTE_CODF")VALUES(?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5) [22007-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:860)
	at org.h2.table.Column.convert(Column.java:148)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:157)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:143)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:322)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:2966)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1085)
	at org.h2.server.web.WebApp.process(WebApp.java:227)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NumberFormatException: For input string: "mar"
	at java.lang.NumberFormatException.forInputString(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:278)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 16 more
02-16 22:10:14 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Cannot parse "DATE" constant "01-mar-2016"; SQL statement:
INSERT INTO "PUBLIC"."TERAPIA"("ID","DESCRIZIONE","DATAINIZIO","DATAFINE","PAZIENTE_CODF")VALUES(?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5) [22007-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:860)
	at org.h2.table.Column.convert(Column.java:148)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:157)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:143)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:322)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:2966)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1085)
	at org.h2.server.web.WebApp.process(WebApp.java:227)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.NumberFormatException: For input string: "mar"
	at java.lang.NumberFormatException.forInputString(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:278)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 16 more
02-16 22:11:30 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'' (TERAPIA: ID INT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TERAPIA"("ID","DESCRIZIONE","DATAINIZIO","DATAFINE","PAZIENTE_CODF")VALUES(?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5) [22018-187]
02-16 22:11:30 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'' (TERAPIA: ID INT NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TERAPIA"("ID","DESCRIZIONE","DATAINIZIO","DATAFINE","PAZIENTE_CODF")VALUES(?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5) [22018-187]
02-16 22:11:39 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Cannot parse "DATE" constant ; SQL statement:
INSERT INTO "PUBLIC"."TERAPIA"("ID","DESCRIZIONE","DATAINIZIO","DATAFINE","PAZIENTE_CODF")VALUES(?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5) [22007-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:860)
	at org.h2.table.Column.convert(Column.java:148)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:157)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:143)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:322)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:2966)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1085)
	at org.h2.server.web.WebApp.process(WebApp.java:227)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.StringIndexOutOfBoundsException: String index out of range: 0
	at java.lang.String.charAt(Unknown Source)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:267)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 16 more
02-16 22:11:39 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Cannot parse "DATE" constant ; SQL statement:
INSERT INTO "PUBLIC"."TERAPIA"("ID","DESCRIZIONE","DATAINIZIO","DATAFINE","PAZIENTE_CODF")VALUES(?,?,?,?,?) -- (?1, ?2, ?3, ?4, ?5) [22007-187]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.value.ValueDate.parse(ValueDate.java:81)
	at org.h2.value.Value.convertTo(Value.java:860)
	at org.h2.table.Column.convert(Column.java:148)
	at org.h2.command.dml.Insert.insertRows(Insert.java:143)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:78)
	at org.h2.command.Command.executeUpdate(Command.java:254)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:157)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:143)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:322)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:2966)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1085)
	at org.h2.server.web.WebApp.process(WebApp.java:227)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:137)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.StringIndexOutOfBoundsException: String index out of range: 0
	at java.lang.String.charAt(Unknown Source)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:267)
	at org.h2.value.ValueDate.parse(ValueDate.java:79)
	... 16 more
02-16 22:16:23 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT * FROMTERAPIA[*]"; SQL statement:
SELECT * FROMTERAPIA [42000-187]
